File: easy/2239/csharp/Solution.cs
using System;

public class Solution
{
    public int FindClosestNumber(int[] nums)
    {
        int closest = nums[0];
        foreach (int num in nums)
        {
            if (Math.Abs(num) < Math.Abs(closest) || (Math.Abs(num) == Math.Abs(closest) && num > closest))
            {
                closest = num;
            }
        }
        return closest;
    }
}

File: easy/2239/c/Solution.c
#include<stdio.h>
#include<stdlib.h>

int findClosestNumber(int* nums, int numsSize) {
    int closest = nums[0];
    for (int i = 1; i < numsSize; i++) {
        if (abs(nums[i]) < abs(closest) || 
            (abs(nums[i]) == abs(closest) && nums[i] > closest)) {
            closest = nums[i];  
        }
    }
    return closest;
}